### DISCUSSION TITLE
Is this completely self-hosted?

### DISCUSSION THREAD

**Discussion by Gandalf-the-Blue** [hayabhay/whisper-ui#42] [Q&A] [answered]
Hi,
Intrigued by this project. Just one question - while it uses OpenAI's Whisper model, does it download the model and then run everything locally, or does it send out any audio/text data to OpenAI servers or other servers for processing?

**Comment by hayabhay** [comment, marked as answer]
Yes. It is completely self hosted. No data is transmitted outside. [Even Streamlit's telemetry is disabled.](https://github.com/hayabhay/whisper-ui/blob/ee4bc894b4978e0406b2a0f8de3bc0d1e5d3f0b2/app/.streamlit/config.toml#L2)
